src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -59,6 +60,8 @@ public PersonCard(Person person, int displayedIndex) {
         address.setText(person.getAddress().value);
         email.setText(person.getEmail().value);
         birthday.setText(person.getBirthday().toUi());
+        debt.setText("Debt: " + person.getDebt().value.toString());
```

Comment:
Debt probably has to be formatted with `$` sign and 2 dp. Could do `debt.setText("Debt: " + person.getDebt().toUi())`

Reply:
`toString()` could be used for debugging etc

Reply:
Sounds like a good idea here.